README.md

```diff
@@ -448,11 +448,10 @@ Then use the "Launch new AppServer" button in the web UI to provision a server
 with the updated settings, and click "Activate this app server" to use the new
 server when it's ready.
 
-**To delete an instance**, ensuring that all virtual machines are terminated, run:
+**To delete an instance**, which will also terminate all associated AppServers
+and their virtual machines, run:
 
 ```python
-for appserver in instance.appserver_set.all():
-    appserver.terminate_vm()
```

Comment:
@bradenmacdonald I think it would be worth mentioning in the README that it's possible to terminate VMs associated with an instance, but keep its app servers (and the instance itself) around. Maybe just add another section that starts like this:

```
**To terminate all VMs associated with an instance** ...
```

... and add this code for terminating VMs back in.